opendcs-web-client/src/main/webapp/resources/jsp/includes/decodes.jsp

```diff
@@ -64,6 +63,13 @@
 <script src="../resources/js/lib/opendcs_utilities.js"></script>
 <%-- /opendcs web files --%>
 
+<%-- Get API Path --%>
+<script>
+	window.API_BASE_URL = '<%= getServletConfig().getServletContext().getInitParameter("api_base_url") %>';
```

Comment:
Based on above comment, basically keep this logic but make it a clientside grab of the existing window.location and change the base path.

Reply:
Sounds good - updated.